src/ssvtools/modify_coordinates.py

```diff
@@ -0,0 +1,239 @@
+    :param coordinate_field_name: name of the chosen coordinate field for SSV.
+    :param template_region: template region where the template vagus scaffold is loaded.
+    :param template_coordinate_field_name: name of the coordinate field for template vagus scaffold.
+    :param trunk_group_name: name of trunk group.
+    :param unit_conversion_factor: Factor to bring SSV into the same scale as template scaffold. If none, mesh integral
+    of the respective trunk lengths will be used to calculate the required scale factor.
+    """
+    fieldmodule = region.getFieldmodule()
+    mesh3d = fieldmodule.findMeshByDimension(3)
+    fieldcache = fieldmodule.createFieldcache()
+    coordinates_field = fieldmodule.findFieldByName(coordinate_field_name).castFiniteElement()
+    tmp_coordinates_field = fieldmodule.findFieldByName(template_coordinate_field_name).castFiniteElement()
+
+    vagus_trunk_group = get_vagus_trunk_group(fieldmodule)
+    trunk_mesh_group3d = vagus_trunk_group.getMeshGroup(mesh3d)
+    elements_along_data_trunk = trunk_mesh_group3d.getSize()
+
+    template_fieldmodule = template_region.getFieldmodule()
+    template_mesh3d = template_fieldmodule.findMeshByDimension(3)
+    vagus_trunk_group = get_vagus_trunk_group(template_fieldmodule)
+    trunk_mesh_group3d = vagus_trunk_group.getMeshGroup(template_mesh3d)
+    elements_along_template_trunk = trunk_mesh_group3d.getSize()
+    assert elements_along_template_trunk == elements_along_data_trunk, \
+        "adopt_template_trunk_coordinates. Number of trunk elements in data does not match number of trunk elements in " \
+        "template vagus."
+
+    # Get radius from data trunk
+    elem_iter = mesh3d.createElementiterator()
+    element = elem_iter.next()
+    element_id = element.getIdentifier()
+    bd2_mag_list = []
+    bd3_mag_list = []
+    bd12_mag_list = []
+    bd13_mag_list = []
+    ln = 2
+
+    # Loop through SSV trunk elements to store radius information
+    while element.isValid() and element_id <= elements_along_template_trunk:
+        eft = element.getElementfieldtemplate(coordinates_field, -1)
+        if element_id == 1:
+            ln = [1, 2]
+        else:
+            ln = [2]
+        for i in range(len(ln)):
+            node = element.getNode(eft, ln[i])
+            fieldcache.setNode(node)
+            _, bd2 = coordinates_field.getNodeParameters(fieldcache, -1, Node.VALUE_LABEL_D_DS2, 1, 3)
+            _, bd3 = coordinates_field.getNodeParameters(fieldcache, -1, Node.VALUE_LABEL_D_DS3, 1, 3)
+            _, bd12 = coordinates_field.getNodeParameters(fieldcache, -1, Node.VALUE_LABEL_D2_DS1DS2, 1, 3)
+            _, bd13 = coordinates_field.getNodeParameters(fieldcache, -1, Node.VALUE_LABEL_D2_DS1DS3, 1, 3)
+            bd2_mag_list.append(magnitude(bd2))
+            bd3_mag_list.append(magnitude(bd3))
+            bd12_mag_list.append(magnitude(bd12))
+            bd13_mag_list.append(magnitude(bd13))
```

Comment:
I think it's fine to use the magnitude for bd2, bd3, but I don't think you can for bd12 and bd13 because it eliminates the sign.
Imagine the template is getting thicker in xi1, but the SSV is getting thinner: the vector direction would have to be reversed. I'll think a bit about how to do this & we can talk about it.